jib-core/src/main/java/com/google/cloud/tools/jib/cache/LayerMetadata.java

```diff
@@ -43,10 +43,6 @@ static LayerMetadata from(List<Path> sourceFiles, FileTime lastModifiedTime) {
   private final FileTime lastModifiedTime;
 
   LayerMetadata(List<String> sourceFiles, FileTime lastModifiedTime) {
-    if (sourceFiles.isEmpty()) {
```

Comment:
Would allow possibly invalid apps through? All resources, no source? I mean I guess we don't need to care if the user is being insane.

Reply:
Yeah, it would allow that, though I think our guarantee could be that the application runs in the container like it would locally.